weblate/templates/accounts/email-sent.html

```diff
@@ -27,19 +27,19 @@ <h4 class="panel-title">
   <div class="panel-body">
 
     {% if is_reset %}
-      <p>{% blocktrans %}If your e-mail address exists in our database, you will receive further password recovery instructions to your inbox.{% endblocktrans %}</p>
```

Comment:
Invoking "database" and who owns it complicates matters.
"Inbox" is unreferenced.

Reply:
The important thing is that user will not receive anything in case the e-mail is not registered. And the feedback is not provided here for security reasons.

Reply:
But the better solution is to send e-mail to the affected user in this case, I've done that in 77f9e7078bd5f357e1e0bc0a5eddbeb16fdf01e4.